config/sidebar.ts

```diff
@@ -41,7 +45,7 @@ export function makeSidebar(): StarlightSidebarConfig {
 			}
 			currentSubGroup = newGroup;
 		} else {
-			currentSubGroup.items.push({
+			currentSubGroup?.items.push({
```

Comment:
And won’t need to handle undefined types here:
```suggestion
			currentSubGroup.items.push({
```